test/e2e/specs/__snapshots__/undo.test.js.snap

```diff
@@ -13,3 +13,39 @@ exports[`undo Should undo to expected level intervals 1`] = `
 <p>test</p>
 <!-- /wp:paragraph -->"
 `;
+
+exports[`undo should undo typing after arrow navigation 1`] = `
+"<!-- wp:paragraph -->
+<p>before keyboar after keyboardd</p>
```

Comment:
The result here _looks_ off, but I guess the test was written in a way where it's expected. Might catch a future maintainer off-guard.

Reply:
I'll investigate. This seems wrong.